contracts/src/interfaces/ITeleporterTokenBridge.sol

```diff
@@ -17,7 +17,8 @@ import {ITeleporterReceiver} from "@teleporter/ITeleporterReceiver.sol";
  * @param destinationBlockchainID blockchainID of the destination
  * @param destinationBridgeAddress address of the destination token bridge instance
  * @param recipient address of the recipient on the destination chain
- * @param primaryFee amount of tokens to pay for Teleporter fee on the source chain
+ * @param feeTokenAddress address of the ERC20 contract to pay for Teleporter fee
+ * @param primaryFee amount of tokens to pay for Teleporter fee
```

Comment:
```suggestion
 * @param feeTokenAddress address of the ERC20 contract to optionally pay a Teleporter message fee
 * @param primaryFee amount of tokens to pay as the optional Teleporter message fee
```

Nit: I'd like to try to avoid ever using the phrase "Teleporter fee" because it makes it seem like the Teleporter protocol itself charges fees, which it does not, or that the fees are required to be paid. Would be good to do a pass over the contracts/docs and see if we use it anywhere else.